scripts/start-preview

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
do we even need this file?

Reply:
Yes. Maintaining scripts over all packages is tedious. I would extract even more package scripts to the `scripts/` folder so that we don't have to duplicate them